service/core.go

```diff
@@ -126,25 +126,25 @@ func (s *serviceCore) AuthClient() auth.Client {
 }
 
 func (s *serviceCore) Name() string {
```

Comment:
Based on discussion with the team, decided to get rid of all application* and rename yaml keys to service*. It will be serviceName. Expect an update soon.